media/img/firefox/features/pip/pip-cook.svg

```diff
@@ -0,0 +1,26 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 152 134" width="152" height="134">
+  <style>
+    .st3{fill:#d8d8d8}.st4{fill:none;stroke:#d0d0d0;stroke-width:2.1192;stroke-linecap:round;stroke-linejoin:round}
+  </style>
+    <path fill="#80808e" d="M142.3 1.3H3.4c-.9 0-1.7.7-1.7 1.6v105.6c0 .9.7 1.7 1.7 1.7h138.9c.9 0 1.7-.7 1.7-1.7V2.9c0-.9-.8-1.6-1.7-1.6z"/>
+    <path fill="#eeeef1" d="M1.7 25.2v83.3c0 .9.7 1.7 1.7 1.7h138.9c.9 0 1.7-.7 1.7-1.7V25.2H1.7z"/>
+    <path fill="#9f9fad" d="M37.8 9.6V3.5H15.1v6.1H1.7v15.6h142.2V9.6z"/>
+    <path d="M26.5 13.9h109.4v6.7H26.5z" class="st3"/>
+    <path d="M26.9 81.7h16.4" class="st4"/>
+    <path d="M26.9 76h31.6" class="st4"/>
+    <path d="M26.9 70.4h31.6" class="st4"/>
+    <path d="M26.9 64.7h31.6" class="st4"/>
+    <path d="M18.3 70.4h.8" class="st4"/>
+    <path d="M18.3 64.7h.8" class="st4"/>
+    <path d="M18.3 76h.8" class="st4"/>
+    <path d="M18.3 81.9h.8" class="st4"/>
+    <path d="M18.3 32h109v22h-109z" class="st3"/>
+    <path fill="#028fed" d="M70.5 64.6H150c.9 0 1.7.7 1.7 1.7v50c0 .9-.7 1.7-1.7 1.7H70.5c-.9 0-1.7-.7-1.7-1.7v-50c.1-.9.8-1.7 1.7-1.7z"/>
+    <path fill="#ecf0f1" d="M124.1 97c0-3.3-2.7-6.1-6.1-6.1s-6 2.7-6 6.1c-.1.1 12.1 0 12.1 0z"/>
+    <path fill="#3fe1b0" d="M102.7 89.8C98.3 87.2 94 86.4 93 88c-1 1.6 1.8 5 6.2 7.7 4.4 2.6 9.7 1.8 9.7 1.8s-1.8-5.1-6.2-7.7z"/>
+    <path fill="#15c4a2" d="M93.1 88c-1 1.6 1.8 5 6.2 7.7 4.4 2.6 9.7 1.8 9.7 1.8L93.1 88z"/>
+    <path fill="#f3d55b" d="M122.1 70.3l3.1 3.1-3.1 3.1-3.1-3.1z"/>
+    <path fill="#ffa266" d="M112 77.5l5.4 5.4-5.4 5.4-5.4-5.4z"/>
+    <path fill="#591dce" d="M85.4 98.7v3.4h-3.7c-.4 0-.8.3-.8.8v2.7c0 1.9 1.5 3.4 3.4 3.4h1.2c.7 12.9 4.3 14.6 24.8 14.6 24.9 0 24.9-2.5 24.9-24.8 0-.1-49.8 0-49.8-.1z"/>
+    <path fill="#7542e5" d="M110.3 123.5c20.4 0 24.1-1.7 24.8-14.6h1.2c1.9 0 3.4-1.5 3.4-3.4v-2.7c0-.4-.3-.8-.8-.8h-3.7v-3.4h-24.9v24.9z"/>
+</svg>
```

Comment:
I don't think we have any lint running on SVG files, but GitHub added this 🔴 icon at the end of this file: 
<img width="229" alt="image" src="https://user-images.githubusercontent.com/2692333/109687882-5000f080-7b49-11eb-8fd4-8cebe09d282f.png">

Feel free to disregard.

Reply:
Added blank lines at the ends of these files, just to be safe (and make GitHub happy).